lua/weapons/weapon_snowball.lua

```diff
@@ -68,6 +68,7 @@ function SWEP:PrimaryAttack()
 			local front = self.Owner:GetAimVector()
 			ball:SetOwner(self.Owner)
 			ball:SetPos(self.Owner:GetShootPos() + front * 10 + self.Owner:EyeAngles():Up() * -5)
+			Ball:SetColor(self.Owner:GetNWVector("SnowballColor"), Vector(1, 1, 1)):ToColor)
```

Comment:
you're missing parens after :ToColor and you have an extra close paren after "SnowballColor"

Reply:
Okay this should now be fixed